.github/workflows/test.yaml

```
@@ -34,7 +34,10 @@ jobs:
       # - name: Install the project
       #   run: uv sync --all-extras --dev
       - name: Install the project
-        run: uv sync --extra dev
+        runs: |
```

actionlint (1.7.4)

unexpected key "runs" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)